modules/local/samplesheet_check.nf

```diff
@@ -0,0 +1,31 @@
+// Import generic module functions
+include { saveFiles } from './functions'
```

Comment:
Difference between this and `check_samplesheet.nf`?

Reply:
There is not much difference. I will keep `samplesheet_check.nf` because it is from the template update

Reply:
:+1: